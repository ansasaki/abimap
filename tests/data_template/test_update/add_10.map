# This map file was created with PROGRAM_NAME_VERSION

LIBTC1_1_9_0
{
    global:
        another_symbol;
    local:
        *;
} ;


