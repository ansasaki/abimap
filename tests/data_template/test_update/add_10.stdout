Added:
    symbol

# This map file was updated with PROGRAM_NAME_VERSION

LIBTC1_1_9_0
{
    global:
        another_symbol;
    local:
        *;
} ;

LIBTC1_1_10_0
{
    global:
        symbol;
} LIBTC1_1_9_0;

